resolwe_bio/tools/pathway_predictor.py

```diff
@@ -1,7 +1,26 @@
 #!/usr/bin/env python3
+import sys
+import json
+from subprocess import call
 from cameo import models
 from cameo.strain_design import pathway_prediction
 
-model = models.bigg.iMM904
+RESULTS = []
+
+
+def reaction_to_dict(reaction):
+    return dict(
+        id=reaction.id,
+        name=reaction.name,
+        reaction_string=reaction.build_reaction_string(use_metabolite_names=True),
+    )
+
+
+def show_results(pathway):
+    RESULTS.append([reaction_to_dict(reaction) for reaction in pathway.reactions])
+    call(['re-save', 'pathways', json.dumps(RESULTS)])
```

Comment:
[This ](https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/tools/geneset2storage.py#L30) is an example of code that would create an output json file to be saved with `re-save` in the .yml file